week1/homework/src/server.js

```diff
@@ -10,6 +10,32 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    switch (request.url) {
+      case '/state':
+        response.setHeader('Content-Type', 'application/json');
+        response.write(`{"state": ${state}}`);
```

Comment:
I can tell you understand what a JSON string is, that's the most important, but we have a nice little method for making a JSON string out of a JS object: [`JSON.stringify()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).

Reply:
I'm well aware of that but the code was short so i though it's not necessary, if you insist i'll update the code.